io_scene_godot/structures.py

```diff
@@ -283,30 +283,31 @@ def to_string(self):
         )
 
 
-# ------------------ Implicit Conversions of Blender Types --------------------
-def mat4_to_string(mtx):
-    """Converts a matrix to a "Transform" string that can be parsed by Godot"""
-    def fix_matrix(mtx):
-        """ Shuffles a matrix to change from y-up to z-up"""
-        # Todo: can this be replaced my a matrix multiplcation?
-        trans = mathutils.Matrix(mtx)
-        up_axis = 2
+def fix_matrix(mtx):
+    """ Shuffles a matrix to change from y-up to z-up"""
+    # Todo: can this be replaced my a matrix multiplcation?
```

Comment:
I would make this my own commit except that I know you've made this edit here. But fix_matrix can be replaced by:

```python
def fix_matrix(mtx):
    """ Shuffles a matrix to change from y-up to z-up"""
    new_mat = mtx * mathutils.Matrix([
        [1, 0, 0, 0],
        [0, 0, -1, 0],
        [0, 1, 0, 0],
        [0, 0, 0, 1],
    ])
    new_mat[1], new_mat[2] = new_mat[2].copy(), -new_mat[1].copy()
    return new_mat
```

Reply:
Yeah, but in master branch it's in a different place, and when this gets merged the change may go unnoticed.

Reply:
then leave it after this PR merged, it is not urgent